jdi-light-angular-tests/src/main/java/io/github/com/entities/Users.java

```diff
@@ -0,0 +1,13 @@
+package io.github.com.entities;
+
+import com.epam.jdi.tools.DataClass;
+
+public class Users extends DataClass<Users> {
```

Comment:
Please rename Users to User. Entity class that describes a single entity should have singular value name.